userspace/libsinsp/sinsp_filtercheck_thread.cpp

```diff
@@ -280,6 +286,28 @@ int32_t sinsp_filter_check_thread::parse_field_name(const char* str, bool alloc_
 
 		return res;
 	}
+	else if(STR_MATCH("proc.concat_aname"))
+	{
+		m_field_id = TYPE_CONCAT_ANAME;
+		m_field = &m_info.m_fields[m_field_id];
+
+		int32_t res = 0;
+
+		try
+		{
+			res = extract_arg("proc.concat_aname", val, NULL);
+		}
+		catch(...)
+		{
+			if(val == "proc.concat_aname")
+			{
+				m_argid = -1;
+				res = (int32_t)val.size();
+			}
+		}
+
+		return res;
```

Comment:
Can we find a way to reuse this code maybe with a helper function :thinking: ? It seems duplicated in many places